sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md

```diff
@@ -144,16 +153,70 @@ directive:
   - remove-operation: GenerateDetailedCostReportOperationResults_Get
   - remove-operation: GenerateDetailedCostReportOperationStatus_Get
   - remove-operation: Operations_List
+#   - remove-operation: CostAllocationRules_CheckNameAvailability
 
   # Could not set ResourceTypeSegment for request path /{scope}
-  - from: scheduledActions.json
-    where: $.parameters.scopeParameter
-    transform: $['x-ms-skip-url-encoding'] = true;
-  - from: costmanagement.json
-    where: $.parameters.scopeViewParameter
-    transform: $['x-ms-skip-url-encoding'] = true;
+  - from: openapi.json
+    where: $.paths['/{scope}/providers/Microsoft.CostManagement/scheduledActions'].get
+    transform: >
+        $['parameters'][1]['x-ms-skip-url-encoding'] = true;
```

Comment:
I am not sure the exact details of the policy of our typespec migration - but I think this should be an issue fixed on our typespec migration.
Or at least this would be an issue that we would meet in the future when we switch to use typespec to generate this RP.